app/assets/stylesheets/views/_covid.scss

```diff
@@ -313,3 +279,22 @@ $c19-landing-page-header-background: #272828;
 .covid-timeline__item-heading {
   margin-bottom: govuk-spacing(1);
 }
+
+.covid__logo-wrapper {
+  margin-bottom: govuk-spacing(6);
+  text-align: center;
+
+  @include govuk-media-query($from: tablet) {
+    text-align: left;
+  }
+}
+
+.covid__logos {
+  max-width: 100%;
+  height: auto;
+  margin-top: govuk-spacing(4);
+
+  @include govuk-media-query($from: tablet) {
+    max-height: 45px; // the SVG canvas height is too big on IE
```

Comment:
Is this an IE bug or something we can change in the SVG? I notice we specify canvas height of 45 in the SVG already. It'd be useful to have some more info in the commit message about the IE problem, for example source, versions affected.

Reply:
I believe it affects all IE versions - I will confirm shortly.

Reply:
## IE11

### Without max-height
<img width="1134" alt="Screenshot 2021-01-05 at 09 42 57" src="https://user-images.githubusercontent.com/4599889/103631080-77b54d00-4f3a-11eb-9405-54ee530cdfe4.png">

### With max-height
<img width="1130" alt="Screenshot 2021-01-05 at 09 43 10" src="https://user-images.githubusercontent.com/4599889/103631105-7be16a80-4f3a-11eb-9ee7-8d34422223f1.png">

Reply:
Just checked all the other IE versions - same issue